frontend/packages/console-app/src/components/guided-tours/QuickStartSection.scss

```diff
@@ -0,0 +1,46 @@
+.co-quick-start-task {
+    &__title {
+        display: inline-block;
+        padding-right: var(--pf-global--spacer--md) !important;
+        color: var(--pf-global--primary-color--100) !important;
+    }
+    &__title-success {
+        color: var(--pf-global--palette--green-500) !important;
+    }
+    &__title-failed {
+        color: var(--pf-chart-global--danger--Color--100) !important;
+    }
+    &__task-icon {
+        border-radius: var(--pf-global--BorderRadius--lg);
+        background-color: var(--pf-global--palette--blue-400) !important;
+        color: var(--pf-global--Color--light-100);
+        display: inline-block;
+        flex-shrink: 0;
+        font-size: var(--pf-global--icon--FontSize--md);
+        line-height: var(--pf-global--FontSize--xl);
+        margin-right: var(--pf-global--spacer--sm);
+        min-width: var(--pf-global--FontSize--xl);
+        padding: var(--pf-global--BorderWidth--sm) var(--pf-global--BorderWidth--lg);
+        text-align: center;
+        white-space: nowrap;
```

Comment:
Do you really need so much of css for task icon?

Reply:
Plus you're not even using the task icon in this component. The css should be colocated with the component it's being used in.

Reply:
Actually most of the css in this file is not used in `QuickStartSection` component.

Reply:
moved to separate scss files according to usage in components